src/api/app/models/token/release.rb

```diff
@@ -10,6 +10,14 @@ def call(options)
     manual_release_targets = package_to_release.project.release_targets.where(trigger: 'manual')
     raise NoReleaseTargetFound, "#{package_to_release.project} has no release targets that are triggered manually" unless manual_release_targets.any?
 
+    # first a permission check to avoid incomplete releases
+    manual_release_targets.each do |release_target|
+      unless User.session!.can_modify?(release_target.target_repository.project)
+        raise CmdExecutionNoPermission, "no permission to write in project #{release_target.target_repository.project.name}"
```

Comment:
`CmdExecutionNoPermission` throws an uninitialized constant error, will have a look

Reply:
okay this is addressed